src/lavinmq/amqp/queue/message_store.cr

```diff
@@ -200,6 +200,23 @@ module LavinMQ
         count
       end
 
+      def purge_all
+        @segments.each_value { |f| delete_file(f) }
+        @segments.clear
```

Comment:
Probably a good idea to assign a new hash instead. `Hash#clear` doesn't shrink if it was large. Same for all other `clear` in this method